users/models/identity.py

```diff
@@ -429,3 +431,17 @@ def generate_keypair(self):
         self.private_key, self.public_key = RsaKeys.generate_keypair()
         self.public_key_id = self.actor_uri + "#main-key"
         self.save()
+
+    ### Config ###
+
+    @cached_property
+    def config_identity(self) -> Config.IdentityOptions:
+        return Config.load_identity(self)
+
+    def lazy_config_value(self, key: str):
+        """
+        Lazily load a config value for this Identity
+        """
+        if key not in Config.IdentityOptions.__fields__:
+            raise KeyError(f"Undefined IdentityOption for {key}")
+        return lazy(lambda: getattr(self.config_identity, key))
```

Comment:
I'm not sure if this lazy method makes sense. The tight coupling to an Identity means we probably can't use it during Form setup like we've done with Config.system values.